### DISCUSSION TITLE
How to get a message without a slash?

### DISCUSSION THREAD

**Discussion by huyhqit** [yagop/node-telegram-bot-api#1085] [Q&A] [answered]
I want to get a message from another, but onText method only gets a message with a command. How do I get a message without a slash?
Thank you

**Comment by danielperez9430** [comment, marked as answer]
You can use also bot on text, but all depends of the **regex** that you pass to the command.

`bot.onText("/hello/") ` // Example that catch messages that contains "hello"

or 

```js
// Catch all the messages
bot.on("message", (msg) => {
 // Handle the text inside the msg object and you 
 if(msg?.text && msg.text.includes("hello")) {
  // Do something
 }
})
```
